webapp/components/abstract-test-file-results-table.html

```diff
@@ -8,24 +8,23 @@
   <script>
     // eslint-disable-next-line no-unused-vars
     const AbstractTestFileResultsTable = superClass => class extends superClass {
-      static get is() {
-        return 'abstract-test-file-results-table';
-      }
-
       static get properties() {
         return {
           subtestNames: {
             type: Array,
             value: [],
           },
+          hasSubtests: {
+            type: Boolean,
+          },
         };
       }
 
-      subtestNameForDisplay(subtestName, isTestHarness) {
+      subtestNameForDisplay(subtestName, hasSubtests) {
         if (subtestName !== 'STATUS') {
           return subtestName;
         }
-        return isTestHarness ? 'Harness status' : 'File status';
+        return hasSubtests ? 'Harness status' : 'File status';
```

Comment:
Wasn't this fixed to "Test status" in this PR already?

Reply:
Sorry... I have too many local branches and confused myself when merging. Thanks for double checking on this!